src/site/_includes/path.njk

```diff
@@ -2,7 +2,9 @@
 layout: layout
 ---
 
-{% set topics = path.topics | removeDrafts(lang) %}
+{% set path = paths[pathName] %}
+{% set topics = path.topics  %}
+{# {% set topics = path.topics | removeDrafts(lang) %} #}
```

Comment:
Should this comment be removed?
Also, I think the behavior is that it used to remove whole topic sections if every item in the section was a draft. Does this new code preserve that behavior?